recipients_app/lib/data/repositories/organization_repository.dart

```diff
@@ -0,0 +1,26 @@
+import "package:app/data/models/organization.dart";
+import "package:cloud_firestore/cloud_firestore.dart";
+
+class OrganizationRepository {
+  final FirebaseFirestore firestore;
+
+  const OrganizationRepository({
+    required this.firestore,
+  });
+
+  /// Fetches the user data by userId from firestore and maps it to a recipient object
```

Comment:
Seems like copy paste from UserRepository, can we update the comment? (Or remove)

Reply:
done